test/ecto/adapters/postgres_test.exs

```diff
@@ -45,7 +45,7 @@ defmodule Ecto.Adapters.PostgresTest do
   end
 
   defp normalize(query) do
-    {query, _params} = Ecto.Query.Planner.prepare(query, [])
+    {query, _params} = Ecto.Query.Planner.prepare(query, [], %{})
```

Comment:
@mobileoverlord You need to update your call to Planner.prepare (if you do any).